functional-tests/README.md

```diff
@@ -51,8 +53,35 @@ Only on windows:
 
 Only on MacOS:
 
-    ./gradlew safariTest -DsafariTest.single=CustomJUnitSpecRunner
+    DEPRECATED ./gradlew safariTest -DsafariTest.single=CustomJUnitSpecRunner
+
+## Run tests in project root
+
+The tests can also be run from the projects root using the command 'make test'
+
+## Modifying tests that run in the pipeline
+
+The shell script controlling how the tests in the pipeline are run is run_test.sh. If new products need to be added to the
```

Comment:
What are products? 🤔 Maybe test suites? commands? Im not too sure what's a better word

Also I dunno if its github's weird formatting but there's a line break here

Reply:
done